src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -18,26 +17,28 @@ export const plugin: ComputationPlugin = {
 // Include available methods in this array.
 const ABUNDANCE_METHODS = ['median', 'q3', 'variance', 'max'];
 
-// Include known collection variables in this array.
-const ABUNDANCE_COLLECTION_VARIABLES = [
-  { entityId: 'EUPATH_0000808', variableId: 'EUPATH_0009253' },
-];
-
 function variableDescriptorToString(
   variableDescriptor: VariableDescriptor
 ): string {
   return JSON.stringify(variableDescriptor);
 }
 
 export function AbundanceConfiguration(props: ComputationConfigProps) {
-  const [name, setName] = useState('New ranked abundance app');
-  const [collectionVariable, setCollectionVariable] = useState(
-    variableDescriptorToString(ABUNDANCE_COLLECTION_VARIABLES[0])
-  );
+  const [name, setName] = useState('New ranked abundance tool');
   const [rankingMethod, setRankingMethod] = useState(ABUNDANCE_METHODS[0]);
   const { computationAppOverview, addNewComputation } = props;
-  const study = useStudyMetadata();
-  const entities = useStudyEntities(study.rootEntity);
+  const studyMetadata = useStudyMetadata();
+  // Include known collection variables in this array.
+  const collections = useFlattenedCollectionVariables(studyMetadata.rootEntity);
+  if (collections == null || !collections[0])
```

Comment:
I think this can be changed to the following, once the return-value of the callback passed to `flatMap` in `findCollectionVariables` is updated as I suggested.
```suggestion
  if (collections.length === 0)
```